solarsystem:
    celestial_bodies:
        stars:            
            -   name: "Sun"

        planets:
            -   name: "Earth"
                orbit_radius: 4

            -   name: "Jupiter"          
                orbit_radius: 9

            -   name: "Mars"
                orbit_radius: 6

        natural_satellites:
            -   name: "Moon"
                adjacent_planet: "Earth"

            -   name: "Europa"
                adjacent_planet: "Jupiter"
          
            -   name: "Phobos"
                adjacent_planet: "Mars"
          

    rate: 15
      
           